robots/falcon_bravo7_description/urdf/falcon_bravo7.urdf

```diff
@@ -0,0 +1,452 @@
+                        <axis xyz="1 0 0"/>
+                        <origin rpy="0.0 0.0 -1.0472" xyz="0.19 0.31 0.165"/>
+                    </joint>
+
+                    <joint name="2_propeller_joint" type="fixed">
+                        <parent link="falcon"/>
+                        <child link="2_propeller"/>
+                        <axis xyz="1 0 0"/>
+                        <origin rpy="0.0 0.0 -1.0472" xyz="0.165 -0.307 0.165"/>
+                    </joint>
+
+                    <joint name="1_propeller_joint" type="fixed">
+                        <parent link="falcon"/>
+                        <child link="1_propeller"/>
+                        <axis xyz="1 0 0"/>
+                        <origin rpy="0.0 0.0 -2.094395327" xyz="-0.165 -0.307 0.165"/>
+                    </joint>
+
+            <!--BRAVO ARM-->
+            <link name="link1">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/RS2-266.stl"/>
+                    </geometry>
+                    <material name="blue"/>
+                </visual>
+                <inertial>
+                    <mass value="1.25"/>
+                    <origin xyz="-0.075 -0.006 -0.003"/>
+                    <inertia ixx="0.002108" ixy="0.000182" ixz="-0.000015" iyy="0.002573" iyz="-0.000021" izz="0.003483"/>
+                </inertial>
+                <collision>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/RS2-266_convex_hull.stl"/>
+                    </geometry>
+                </collision>
+            </link>
+
+                    <joint name="base_fixed" type="fixed">
+                        <parent link="falcon"/>
+                        <child link="link1"/>
+                        <origin rpy="0 0 3.14" xyz="-0.325 -0.65 -0.09"/>
+                    </joint>
+
+            <link name="link2">
+                <visual>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/RS2-217.stl"/>
+                    </geometry>
+                    <material name="green"/>
+                </visual>
+                <inertial>
+                        <mass value="1.55" />
+                        <inertia ixx="0.011442" ixy="-0.000484" ixz="0.003405" iyy="0.01298" iyz="-0.001265" izz="0.003202"/>
+                        <origin xyz="0.005 -0.001 0.016" />
+                </inertial>
+                <collision>
+                    <geometry>
+                        <mesh filename="package://example-robot-data/robots/falcon_bravo7_description/meshes/RS2-217_convex_hull.stl"/>
+                    </geometry>
+                </collision>
+            </link>
+
+                    <joint name="joint1" type="revolute">
```

Comment:
As far as I understand, most of the Bravo 7's joints are continuous, not revolute. Is that correct?

If so, please add the code accordingly.